### ISSUE TITLE
It is 

### ISSUE THREAD

**Issue by dreamkeep** [UAVenture/ros-setups#3]
hi sjwilks,
     In the file install_ros.sh  , it shall add a command that  'sudo apt-get -y upgrade' under 54 line.

**Comment by SimonWilks** [comment]
@dreamkeep is something not working without adding that extra line? The update command is just to update after adding the two sources on lines 52 and 53. An upgrade for the OS was already performed on line 21 so nothing else should have changed otherwise.

**Comment by dreamkeep** [comment]
@sjwilks yes, if it don't have 'sudo apt-get -y upgrade' under 54 line,it may not work well.
because this command that 'sudo apt-get -y build-dep console-bridge' will excute fail, the console-bridge can't find.

**Comment by SimonWilks** [comment]
@dreamkeep did it actually fail for you and did apt-get upgrade fix it?

The update command should ensure that it is found when build-dep runs later, no? apt-get upgrade only installs new version (and new version dependencies?) not new packages and build-dep should fetch what's required for building. 

I only ask because another person and I have run the installation script without issue. Of course I can add it as it won't hurt, just wait a few seconds worse case.
